modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/GroupUpdateRequestHandler.java

```diff
@@ -129,13 +129,13 @@ public CompletableFuture<Void> handle(DisasterRecoveryManager disasterRecoveryMa
                     .map(NodeWithAttributes::nodeName)
                     .collect(toSet());
 
-            List<CompletableFuture<Void>> tableFuts = new ArrayList<>(request.partitionIds().size());
+            List<CompletableFuture<Void>> assignmentsUpdateFut = new ArrayList<>(request.partitionIds().size());
```

Comment:
```suggestion
            List<CompletableFuture<Void>> assignmentsUpdateFuts = new ArrayList<>(request.partitionIds().size());
```
As this is a collection